module/proto/peggy/v1/genesis.proto

```diff
@@ -70,51 +71,49 @@ option go_package = "github.com/althea-net/peggy/module/x/peggy/types";
 // refer to not submitting a particular message, the third for submitting a different claim
 // for the same Ethereum event
 message Params {
-  option (gogoproto.stringer)  = false;
+  option (gogoproto.stringer) = false;
 
-  string peggy_id                         = 1;
-  string contract_source_hash             = 2;
-  string bridge_ethereum_address          = 4;
-  uint64 bridge_chain_id                  = 5;
-  uint64 signed_valsets_window            = 6;
-  uint64 signed_batches_window            = 7;
-  uint64 signed_claims_window             = 8;
-  uint64 target_batch_timeout             = 10;
-  uint64 average_block_time               = 11;
-  uint64 average_ethereum_block_time      = 12;
-  bytes  slash_fraction_valset            = 13  [
+  string peggy_id                    = 1;
+  string contract_source_hash        = 2;
+  string bridge_ethereum_address     = 4;
+  uint64 bridge_chain_id             = 5;
```

Comment:
```suggestion
  uint64 eip155_chain_id             = 5;
```

Reply:
is this specific to eip155? eip155 was deprecated in geth 1.10, is this a problem?